tailwind.config.js

```diff
@@ -93,24 +74,134 @@ module.exports = {
+        "in-down": "in-down 300ms ease-in-out",
+        "in-up": "in-up 300ms ease-in-out",
+        "in-scale": "in-scale 300ms linear"
+      },
+      borderRadius: {
+        half: "50%"
+      },
+      boxShadow: {
+        0: "0 4px 8px -1px rgba(0, 0, 0, 0.08), 0 2px 4px -1px rgba(0, 0, 0, 0.04)",
+        1: "0 4px 8px -1px rgba(0, 0, 0, 0.08), 0 2px 4px -1px rgba(0, 0, 0, 0.04)",
+        "1-lg": "0 4px 16px 0 rgba(0, 0, 0, 0.08), 0 2px 8px 0 rgba(0, 0, 0, 0.04)",
+        "1-sm": "0 1px 6px -1px rgba(0, 0, 0, 0.16), 0 1px 2px -1px rgba(0, 0, 0, 0.08)",
+        2: "0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08)",
+        "2-lg": "0 12px 32px -2px rgba(0, 0, 0, 0.1), 0 4px 20px 0 rgba(0, 0, 0, 0.08)",
+        "2-sm": "0 2px 12px -4px rgba(0, 0, 0, 0.2), 0 2px 4px -2px rgba(0, 0, 0, 0.16)",
+        'border-bottom': '0 1px 0 var(--calcite-ui-border-3)',
+        'outline-active': '0 0 0 1px var(--calcite-ui-brand)', 
+        'none': 'none',
+      },
+      keyframes: {
+        "in": {
+          "0%": {
+            opacity: 0
+          },
+          "100%": {
+            opacity: 1
+          }
+        },
+        "in-down": {
+          "0%": {
+            opacity: 0,
+            transform: "translate3D(0, -5px, 0)"
+          },
+          "100%": {
+            opacity: 1,
+            transform: "translate3D(0, 0, 0)"
+          }
+        },
+        "in-up": {
+          "0%": {
+            opacity: 0,
+            transform: "translate3D(0, 5px, 0)"
+          },
+          "100%": {
+            opacity: 1,
+            transform: "translate3D(0, 0, 0)"
+          }
+        },
+        "in-scale": {
+          "0%": {
+            opacity: 0,
+            transform: "scale3D(0.95, 0.95, 1)"
+          },
+          "100%": {
+            opacity: 1,
+            transform: "scale3D(1, 1, 1)"
+          }
+        }
+      },
+      opacity: {
+        disabled: "0.5"
+      },
+      spacing: {
+        "0.5": "0.125rem",
+        "2.5": "0.625rem",
+        "3.5": "0.875rem"
+      },
+      transitionProperty: {
+        margin: "margin",
+        color: "color"
+      },
+      transitionTimingFunction: {
+        cubic: "cubic-bezier(0.215, 0.440, 0.420, 0.880)"
+      }
+    }
   },
+  plugins: [
+    ({ addUtilities }) => {
+      const newUtilities = {
+        ".word-break": {
+          "word-wrap": "break-word",
+          "word-break": "break-word"
+        },
+        ".focus-base": {
+          "outline-offset": 0,
+          "outline-color": "transparent",
+          "transition": "outline-offset 100ms ease-in-out, outline-color 100ms ease-in-out"
+        },
+        ".focus-outset": {
+          "outline": "2px solid var(--calcite-ui-brand)",
+          "outline-offset": "2px"
+        },
+        ".focus-inset": {
+          "outline": "2px solid var(--calcite-ui-brand)",
+          "outline-offset": "-2px"
+        }
+        // TODO: focus-box-shadow
```

Comment:
can we add an issue link here?

Reply:
Or maybe just add an issue and link to the code? Just trying to make sure `// todo` aren't left in the code.

Reply:
@jcfranco what do you think? Maybe we should go through all the `// todo` references and just remove them and have issues instead? https://github.com/Esri/calcite-components/search?q=%2F%2F+todo

Reply:
I'll add an issue and remove this comment.

Reply:
#1458